templates/questions/index.html.ep

```diff
@@ -91,6 +92,13 @@ $(document).ready(function() {
 			<img class="containedImg headerImg" src="/img/newQuestion.png" alt='new question' onclick="toggleQuestionForm()">
 		</div>
 		<div class="headerImgContainer accountButtonContainer" onclick="window.location = '/account'">
+			% if ($c->session('id')){
+        	<div class="Name"><%= $user->{first_name}%></div>
+			%}
+			%if ($c->session->{admin} == 1) {
+			<div class="Name">*</div>
```

Comment:
Yeah I like that idea. I'll play around with this and try to find something that looks nice